modules/op-running-as-root-user-namespace.adoc

```diff
@@ -0,0 +1,34 @@
+[id='op-running-as-root-user-namespace_{context}']
+= Running as root in a user namespace
+
+When you run <something> in a user namespace, your container runs as root (user id 0) but has user privileges on the host. {product-title} uses https://cri-o.io/#:~:text=What%20is%20CRI%2DO%3F,as%20the%20runtime%20for%20kubernetes[CRI-O] as its container runtime interface. CRI-O has a workload feature to run a pod in the user namespace.
+
+To run as a user in a namespace with Openshift Pipelines you need to be able to pass the annotations expected by CRIO to get the pods running  user namespaced. You can directly edit the buildah ClusterTask with `oc edit clustertask buildah` to add this to all users or export the ClusterTask as a task to a specific namespace. 
```

Comment:
```suggestion
With {pipelines-shortname}, to run pods in the user namespace, you must pass the annotations that CRIO expects. To add these annotations for all users, you use the `oc edit clustertask buildah` command to edit the `buildah` cluster task. Or, to add the annotations to a specific namespace, you export the cluster task as as task to that namespace. 
```
- Instructions for using the `oc edit clustertask buildah` command are covered in `modules/op-release-notes-1-0.adoc` release notes file. @sounix000 or you  should be copying that information into a regular doc topic.
- Important: This suggestion is only my best guess at your intended meaning, and might be incorrect. You MUST have an SME re-review this content.

Reply:
IMPORTANT: This ^^^ looks like the introduction that precedes a procedure. You must still cover this information in one or more procedures.

Reply:
In case you need to create additional procedure modules, remember to create separate modules for each independent procedure.